src/__tests__/index.test.js

```diff
@@ -56,7 +56,7 @@ describe('resolve', () => {
         const ipfs = { ...mockIpfs, name: { resolve: jest.fn((identifier, options, callback) => callback('foo', {})) } };
         const ipid = await createIpid(ipfs);
 
-        await expect(ipid.resolve(mockDid)).rejects.toThrow('Invalid DID: did:ipid:QmUTE4cxTxihntPEFqTprgbqyyS9YRaRcC8FXp6PACEjFG');
+        await expect(ipid.resolve(mockDid)).rejects.toThrow('Unable to resolve document with did: did:ipid:QmUTE4cxTxihntPEFqTprgbqyyS9YRaRcC8FXp6PACEjFG');
```

Comment:
`with DID`? 

For consistency with other errors, e.g. `'Document "id" must be a valid DID`